lib/charms/mongodb/v1/mongos.py

```diff
@@ -163,11 +163,6 @@ def add_shard(self, shard_name, shard_hosts, shard_port=Config.MONGODB_PORT):
         shard_hosts = [f"{host}:{shard_port}" for host in shard_hosts]
         shard_hosts = ",".join(shard_hosts)
         shard_url = f"{shard_name}/{shard_hosts}"
-        # TODO Future PR raise error when number of shards currently adding are higher than the
```

Comment:
mongodb actually limits the amount of total migrations occuring so there is no need to do this